superset/assets/javascripts/explore/stores/controls.jsx

```diff
@@ -1881,10 +1881,11 @@ export const controls = {
     },
   },
 
-  js_datapoint_mutator: jsFunctionControl(
-    t('Javascript data point mutator'),
-    t('Define a javascript function that receives each data point and can alter it ' +
-      'before getting sent to the deck.gl layer'),
+  js_data_mutator: jsFunctionControl(
```

Comment:
Sorry I had not yet merge this change when I asked you to rebase earlier, now it's merge and you should re-re-rebase